### ISSUE TITLE
Batch download again not working properly.

### ISSUE THREAD

**Issue by Dibakarroy1997** [lap00zza/9anime-Companion#111]
**Select your issue type:** (check at least one)
- [x] Bug
- [ ] Question
- [ ] Suggestion
- [ ] Other (please describe):

**Describe your issue:**

Noticed it yesterday that all link containing "storage.googleapis.com" cannot be downloaded while using batch download while links with "lh3.googleusercontent.com" were working properly. But today I neither of those two are working. Practically I guess they change something again.

**Comment by lap00zza** [comment]
Hi @Dibakarroy1997 ,

1.Will look into it as soon as I get back 😄 
2. Something like #81 ?

**Comment by Dibakarroy1997** [comment]
@lap00zza Yep exactly. I didn't know there was already a suggestion regarding it :sweat_smile:. 

Regarding 1st one will wait.

**Comment by Dibakarroy1997** [comment]
The download is working now. I just checked it. But now file size of each file is doubled just as mentioned in #110 and also F2 server is not there. Maybe they are currently working on something and it might be a better option to wait until they are done with it.
